app/models/like.rb

```diff
@@ -0,0 +1,12 @@
+class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+  belongs_to :post, class_name: 'Post', foreign_key: 'post_id'
+
+  after_create :update_likes_counter
+
+  private
+
+  def update_likes_counter
+    post.update_column(:likes_counter, post.likes_counter.to_i + 1)
+  end
```

Comment:
- [ ] **[OPTIONAL]** You can use `increment!` to auto-update the `likes_counter`

  ```
  def update_likes_counter
    post.increment!(:likes_counter)
  end
  ```